modules/update-openshift-security-context-constraints.adoc

```diff
@@ -15,14 +15,7 @@ Run the commands in this section only if you are using {product-title} with {ocp
 
 .Procedure
 
-* {product-title} 3.0.57.2 introduces changes to the SCCs. If you are upgrading to {product-title} 3.0.57.2 or higher, you must patch the SCCs by running the following command:
-+
-[source,terminal]
-----
-$ oc patch --type merge scc scanner -p '{"priority":0}'
-----
-
-* {product-title} 3.64.0 renames the SCCs. If you are upgrading to {product-title} 3.64.0 or higher, you must delete and reapply the SCCs:
+* {product-title} 3.64.0 renames the SCCs. If you are upgrading from a version below {product-title} 3.64.0, you must delete and reapply the SCCs, otherwise, skip this step:
```

Comment:
Wondering if it's possible to include this single UL step as some sort of context for the task, and then have the nested OL steps as the actual procedure.